lethal-squirrels/Bangazon-Site
# added delete functionality to product and few other layout changes

Ticket: #13,#18,#19

## DESCRIPTION
I changed layout of the products that show up on the home page
The user can only delete a product if it is not on a completed order
When you click on the MyProducts page, only your products should show up

## STEPS TO TEST
1. Clone ajproddelete to a local repo
2. Test functionality
	Click on MyProducts link in nav bar
        Make sure only the products you have added show up on this page
        
        When you try and delete a product, it should only let you delete a product if it has never been 
        on a completed order.

        On the Home Page, the layout should like the other pages for Products with links to Details et 
        al.
